vsutil.py

```diff
@@ -246,15 +246,16 @@ def depth(clip: vs.VideoNode,
                              dither_type=dither_type.value)
 
 
-def _readable_enums(enum: Type[Enum], module: str = 'vsutil') -> List[str]:
+def _readable_enums(enum: Type[Enum]) -> List[str]:
```

Comment:
Considering how the return value of this function is supposed to be used inside a string anyway and to be read by a human, why not return a string from this function in the first place? You'll never use the 'list' aspect of the return value.